### ISSUE TITLE
build failure after adding use_frameworks! :linkage => :static

### ISSUE THREAD

**Issue by amberv0** [TheWidlarzGroup/react-native-video#2766]
# Bug
Context:
After updating RN to 0.69.1 I realized that I also had to update react-native-video to v6 alpha.1 . 
But I noticed a small change in behavior on ios - after loading a new video from local cache the video was on pause, even though the prop `paused=false` didn't change. 

So I read through the whole doc and found the recommendation about adding `use_frameworks! :linkage => :static` to Podfile. But when I tried it, my app stopped building. 
I tried to disable `use_flipper!()`, since the comment in Podfile says that it's not compatible with `use_frameworks` (which is not a good news by itself, since I'd like to keep using Flipper), but the build kept failing. 

I also tried to set `:fabric_enabled => false` with no success.

So basically I have 2 questions:
- what does `use_frameworks! :linkage => :static` and do I really need this? If yes, is it compatible with Flipper or with fabric?
- why does new video ignore the `paused` prop? Is it a bug to report or a feature?

I'm testing on iPhone 13 simulator with iOS 15.4. Sorry for not providing the repro, maybe it'll turn out it's not needed :) And the xcode build error doesn't seem relevant here.

**Comment by hueniverse** [comment]
@nickfujita can you answer this one?

**Comment by amberv0** [comment]
I just tried to reproduce this with a new project. And indeed, seems like it's not working with the new architecture if I add `use_frameworks! :linkage => :static` to Podfile.

Basically, here are the steps:
```
npx react-native init TestRNVideo
cd TestRNVideo
yarn add react-native-video@next
// manually add use_frameworks! :linkage => :static to Podfile as described here https://github.com/react-native-video/react-native-video/blob/master/API.md#ios
cd ios && RCT_NEW_ARCH_ENABLED=1 pod install && ..
yarn ios
```

Build fails with a bunch of "duplicate output file" messages (commenting out the use_flipper changes nothing). Everything is ok though if I don't add `use_frameworks! :linkage => :static` 
In fact, it even seems to not work with the old architecture...

**Comment by prakashbokati** [comment]
https://github.com/react-native-video/react-native-video/issues/2774

**Comment by diego-ensolvers** [comment]
Same issue, in release mode builds fine, but build fails in debug mode.

**Comment by diego-ensolvers** [comment]
Same issue, in release mode builds fine, but build fails in debug mode

**Comment by freeboub** [comment]
Hello, did you clarify this issue ?
Is there a patch needed in react-native-video package ?

**Comment by freeboub** [comment]
@wood1986 @matinzd do you have a point of view on this issue ?

**Comment by wood1986** [comment]
`use_frameworks!` does not support new architecture in general. This is react-native issue not react-native-video. See https://github.com/facebook/react-native/issues/34123

**Comment by freeboub** [comment]
Thank you @wood1986 
Do let's close this ticket !
